modules/local/format_taxresults.nf

```diff
@@ -8,16 +8,14 @@ process FORMAT_TAXRESULTS {
 
     input:
     path(taxtable)
-    path(taxtable_species)
     path(fastafile)
+    val(outfile)
```

Comment:
It would be sort of more obvious and functional when
https://github.com/nf-core/ampliseq/blob/f1abc2010e849903f429a60e130da92b4ad2f8f8/bin/add_full_sequence_to_taxfile.py#L35-L39
would actually use the value coming in here I think. But it works the way you are doing it right now I guess.

Reply:
It works, but you are right. Thanks for the suggestion!